Dockerfile

```diff
@@ -1,20 +1,19 @@
-FROM ministryofjustice/ruby:2.5.3-webapp-onbuild
+FROM httpd:2.4.43-alpine
```

Comment:
how come we need to drop the version?

Reply:
2.4.43 is the latest alpine version available in the docker httpd image. https://hub.docker.com/_/httpd

Reply:
i think we will need to find a way around this. 2.4 is pretty old and we should really be using 2.7.1

Reply:
oh wait! ignore me. i was thinking it was the ruby version 😆 ! this is fine!